Octokit/Helpers/ApiUrls.cs

```diff
@@ -556,60 +556,60 @@ public static Uri Milestones(string owner, string name)
         /// Returns the <see cref="Uri"/> that returns the specified label.
         /// </summary>
         /// <param name="owner">The owner of the repository</param>
-        /// <param name="repo">The name of the repository</param>
-        /// <param name="name">The milestone number</param>
+        /// <param name="name">The name of the repository</param>
+        /// <param name="milestoneNumber">The milestone number</param>
```

Comment:
@dampir apologies, I didn't see the comments being folded up so I assumed it was still to-be-done